src/Parser/Rule/Package.idr

```diff
@@ -20,117 +20,119 @@ EmptyRule = Grammar () Token False
 export
 equals : Rule ()
 equals = terminal "Expected equals"
```

Comment:
For all of these, I am tempted to suggest the addition of a `Eq Token` implementation
in `Parser.Lexer.Package` and then turn them all into `guard (TOK ==)`.

Reply:
Certainly the existing arrangement feels a little less elegant than it could be.